util/__tests__/stringUtils.test.mjs

```diff
@@ -12,4 +15,48 @@ describe('String utils', () => {
   it('getAcronymFromString if the string is empty, it returns NA', () => {
     expect(getAcronymFromString('')).toBe('');
   });
+
+  it('parseRichTextIntoPlainText returns plain text without HTML and JSX tags', () => {
+    const richText = '<p>This is <strong>bold</strong> and <em>italic</em></p>';
+    const result = parseRichTextIntoPlainText(richText);
+    expect(result).toBe('This is bold and italic');
+  });
+
+  it('parseRichTextIntoPlainText replaces Markdown links with their text content', () => {
+    const richText =
+      'Check out [Node.js](https://nodejs.org/en/) for more information.';
+    const result = parseRichTextIntoPlainText(richText);
+    expect(result).toBe('Check out Node.js for more information.');
+  });
+
+  it('parseRichTextIntoPlainText replaces Markdown lists with their content', () => {
+    const richText = '- Item 1\n- Item 2\n- Item 3';
+    const result = parseRichTextIntoPlainText(richText);
+    expect(result).toBe('Item 1\nItem 2\nItem 3');
+  });
+
+  it('parseRichTextIntoPlainText replaces Markdown underscore, bold, and italic with their content', () => {
+    const richText = 'This is _underscore_, **bold**, and *italic*.';
+    const result = parseRichTextIntoPlainText(richText);
+    expect(result).toBe('This is underscore_, **bold**, and *italic.');
```

Comment:
also @ovflowd take a look to that. I'm not regex expert so could you fix the function ?

Reply:
wdym fix the function? The function is working as expected. What parts are failing in your opinion?

Reply:
the `toBe` value is what you expected for function `parseRichTextIntoPlainText` ? It's not should be `This is _underscore_, **bold**, and *italic*.` ?